chef/automate
# support cert by ip for aws

### :nut_and_bolt: Description: What code changed, and why?

need to support cert by ip on aws deployment

### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-2297

### :+1: Definition of Done

### :athletic_shoe: How to Build and Test the Change

following test is done on Aws deployment
- [x] upgrade from curent version to latest changes
      steps
      1. use curent cli to do provision 
      2. deploy
      3. cert rotate for all service a2/cs/pg/os
      4. upgrade to latest changes
      5. cert show (to see cert is reverted back or not)

- [x] upgrade from latest changes another version
      steps
      1. use latest change cli to do provision 
      2. deploy
      3. cert rotate for all service a2/cs/pg/os
      4. upgrade to latest changes
      5. cert show (to see cert is reverted back or not)
- [x] add node
      steps
      1.  add node 
      2. cert show (to see cert is reverted back or not)
- [x] delete node
      steps
      1.  delete node 
      2. cert show (to see cert is reverted back or not)

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
